elasticsearch/tests/test_elasticsearch.py

```diff
@@ -1,5 +1,5 @@
 import json
-import urllib
+import urllib.request
```

Comment:
Since there are no other changes in this file, is this necessary?

Reply:
i believe so- it is relying on some other module importing the correct submodule:

```console
toor@z14-1:~$ python
Python 3.11.6 (main, Oct 23 2023, 22:48:54) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import urllib
>>> urllib.request.urlopen
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'urllib' has no attribute 'request'
>>> 
```

Reply:
the import state is global - so if another module fully imports urllib (and chances are high enough i guess, or this happens inside requests module, or something...) then it works. but its only correct by accident:

anyways:

```
$ python -c 'import this' | grep implicit
Explicit is better than implicit.
```